actor/kvcounter-tinygo/kvcounter_tinygo.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"strconv"
+
+	actor "github.com/wasmcloud/actor-tinygo"
+	httpserver "github.com/wasmcloud/interfaces/httpserver/tinygo"
+	keyvalue "github.com/wasmcloud/interfaces/keyvalue/tinygo"
+)
+
+func main() {
+	actor.RegisterHandlers(httpserver.HttpServerHandler(&KvcounterTinygo{}))
+}
+
+type KvcounterTinygo struct{}
+
+func (e *KvcounterTinygo) HandleRequest(ctx *actor.Context, req httpserver.HttpRequest) (*httpserver.HttpResponse, error) {
+	kv := keyvalue.NewProviderKeyValue()
+
+	newValue, err := kv.Increment(ctx, keyvalue.IncrementRequest{
+		Key: "tinygo:count", Value: 1,
```

Comment:
To make this example like the other one in Rust, we should pull the name of the counter from the request path.

Reply:
Done!